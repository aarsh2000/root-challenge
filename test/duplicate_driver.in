Driver Arm
Trip Arm 01:20 02:30 43.4
Trip Arm 02:50 03:30 33.14
Driver Arm
Trip Arm 04:00 05:00 39.34