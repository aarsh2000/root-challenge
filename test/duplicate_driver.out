Arm: 116 miles @ 41 mph
